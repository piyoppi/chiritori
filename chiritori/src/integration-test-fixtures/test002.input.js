/*
 * Tab indent
 */
async function main() {
	/* < time-limited to="2020-12-31 23:59:59" > */
	console.log('foo')
	/* < /time-limited > */

	console.log('bar')

	/* < time-limited to="2020-12-31 23:59:59" unwrap-block > */
	if (isReleased) {
		console.log('ðŸ“ŒThis code is unconditionally executed after 2020-12-31T23:59:59.999Z [8]');
		const awesomeFeature = new awesomeFeature()
		awesomeFeature.run();
	}
	/* < /time-limited > */
}
