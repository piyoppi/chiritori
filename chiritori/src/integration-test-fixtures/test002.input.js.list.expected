
-------- [ 1 ]  Ready  --------
             _start
      5 |    /* < time-limited to="2020-12-31 23:59:59" > */
      6 |    console.log('foo')
      7 |    /* < /time-limited > */
                                   ‾end
-------- [ 2 ]  Ready  --------
             _start
     11 |    /* < time-limited to="2020-12-31 23:59:59" unwrap-block > */
     12 |    if (isReleased) {
                             ‾end
-------- [ 3 ]  Ready  --------
         _start
     16 |    }
     17 |    /* < /time-limited > */
                                   ‾end
